src/common/filters/all-exceptions.filter.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  ExceptionFilter,
+  Catch,
+  ArgumentsHost,
+  HttpException,
+  HttpStatus,
+  Logger,
+} from '@nestjs/common';
+import { BaseExceptionFilter } from '@nestjs/core';
+import { Request, Response } from 'express';
+
+@Catch()
+export class AllExceptionsFilter
+  extends BaseExceptionFilter
+  implements ExceptionFilter
+{
+  private readonly logger = new Logger(AllExceptionsFilter.name);
+
+  catch(exception: unknown, host: ArgumentsHost): void {
+    const ctx = host.switchToHttp();
+    const response = ctx.getResponse<Response>();
+    const request = ctx.getRequest<Request>();
+
+    let status: number;
+    let message: string | object;
+
+    if (exception instanceof HttpException) {
+      status = exception.getStatus();
+      const exceptionResponse = exception.getResponse();
+
+      message = exceptionResponse;
```

Comment:
thế thì biến `exceptionResponse` ko cần khai báo nữa, 
mà em sẽ gán thẳng giá trị của vế phải cho biến `message` là được, nhỉ?
```suggestion
      message = exception.getResponse();
```